examples/wallet/app/components/StakePoolList.js

```diff
@@ -1,41 +1,72 @@
 // @flow
 import React from 'react';
-import type { PoolId } from '../models';
+import Button from 'react-bootstrap/Button';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import Container from 'react-bootstrap/Container';
+import type { NewDelegation, PoolId, Delegation } from '../models';
+import type { PoolSelectionHandler } from './StakeDelegation';
+import { percentageFromParts } from '../utils/proportionsHelper';
+import ListingRow from './ListingRow';
+import styles from './StakePoolList.scss';
 
 type Props = {
   stakePools: Array<PoolId>,
-  currentDelegation: PoolId,
-  onSelection: (poolId: PoolId) => void
+  newDelegation: NewDelegation,
+  currentDelegation?: Delegation,
+  onSelection: PoolSelectionHandler
 };
 
-export default ({ stakePools, onSelection, currentDelegation }: Props) => {
+export default ({
+  stakePools,
+  onSelection,
+  currentDelegation,
+  newDelegation
+}: Props) => {
   return (
-    <div>
-      <h3>Pools available</h3>
-      <table>
-        <thead>
-          <tr>
-            <th> Pool id </th>
-          </tr>
-        </thead>
-        <tbody>
-          {stakePools &&
-            stakePools.map(poolId => (
-              <tr key={poolId}>
-                <td>{poolId}</td>
-                <td>
-                  {poolId !== currentDelegation ? (
-                    <button type="button" onClick={() => onSelection(poolId)}>
-                      select
-                    </button>
-                  ) : (
-                    'current delegation'
-                  )}
-                </td>
-              </tr>
-            ))}
-        </tbody>
-      </table>
-    </div>
+    <Container>
+      {stakePools &&
+        stakePools.map((poolId: PoolId) => {
+          const activeDelegation = newDelegation[poolId];
+          const rowStyles: { color: string } = activeDelegation
+            ? { color: activeDelegation.color }
+            : {};
+          const newDelegationPercentage: number =
+            newDelegation && percentageFromParts(newDelegation, poolId);
+          const currentDelegationPercentage =
+            currentDelegation && percentageFromParts(currentDelegation, poolId);
+          return (
+            <ListingRow key={poolId}>
+              <Col style={rowStyles} className={styles.poolId} xs={4}>
+                {poolId}
+              </Col>
+              <Col style={rowStyles} xs={2}>
+                {newDelegationPercentage}%
+              </Col>
+              <Col xs={2}>{currentDelegationPercentage}%</Col>
+              <Col xs={3}>
+                <Row className="justify-content-between">
+                  <Button
+                    type="button"
+                    disabled={!newDelegationPercentage}
+                    size="sm"
+                    onClick={() => onSelection(poolId, -1)}
```

Comment:
I'd like to give this `-1` some name like `increment`, but I don't see an easy way